### ISSUE TITLE
Unable to retrieve data

### ISSUE THREAD

**Issue by ChacalCH** [Tautulli/Tautulli#972]
I installed PlexPy on OMV, as soon as I open the website i get the following warning:

```
2017-02-17 21:53:32	WARNING	Unable to retrieve data for get_sync.
2017-02-17 21:53:32	WARNING	PlexPy PlexTV :: Unable to parse XML for get_synced_items.
2017-02-17 21:53:16	ERROR	Unable to retrieve data for get_activity: 'ascii' codec can't decode byte 0xc3 in position 34: ordinal not in range(128)
Traceback (most recent call last): 
    File "/opt/plexpy/plexpy/webserve.py", line 4028, in get_activity 
        result = pms_connect.get_current_activity() 
    File "/opt/plexpy/plexpy/pmsconnect.py", line 996, in get_current_activity 
        session_data = self.get_sessions(output_format='xml') 
    File "/opt/plexpy/plexpy/pmsconnect.py", line 154, in get_sessions 
        output_format=output_format) 
    File "/opt/plexpy/plexpy/http_handler.py", line 89, in make_request 
        logger.warn(u"Failed to access uri endpoint %s with error %s" % (uri, e)) 
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 34: ordinal not in range(128)`
```

Information about my system:
VERSION 1.4.16

PLEXPY CONFIGURATION

Git Branch:	master
Git Commit Hash:	3742f33d08feb6fb15e74af6318185230205006f
Configuration File:	/opt/plexpy/config.ini
Database File:	/opt/plexpy/plexpy.db
Log File:	/opt/plexpy/logs/plexpy.log
Backup Directory:	/opt/plexpy/backups
Cache Directory:	/opt/plexpy/cache
GeoLite2 Database:	Click here to install the GeoLite2 database.
Arguments:	['--quiet', '--daemon', '--nolaunch', '--pidfile=/var/run/plexpy/plexpy.pid', '--datadir=/opt/plexpy', '--port=8182', '--config=/opt/plexpy/config.ini']
Platform:	Linux 4.9.0-0.bpo.1-amd64
Python Version:	2.7.9 (default, Jun 29 2016, 13:08:31) [GCC 4.9.2]

**Comment by JonnyWong16** [comment]
Do you have a server/user/library/device/etc. with any special characters in the name?

**Comment by ChacalCH** [comment]
Not that I know of.. some libraries have ö/ä/ü characters, but it wasn't a problem before.

**Comment by JonnyWong16** [comment]
I don't know. Double check your server connection settings.

**Comment by Izzu** [comment]
I've also been having the same issue. I'll post my log tomorrow after I get off of work.

**Comment by haxcop** [comment]
Same issue suddenly here

**Comment by pittsmaster** [comment]
Same issue here. Stopped working on the 12th of Feb. Was it a PlexPass update to Plex that broke it?

Plex Media Server version 1.4.2.3400 
Plexpy version 1.4.16

**Comment by DKetch88** [comment]
Same thing here. PlexPy was working great last few months, Then I updated my Plex to newest Version 1.4.3.3433 just a few days ago. Now my PlexPy doesn't work. Anyone find a fix??


WARNING :: CP Server Thread-3 : Unable to retrieve data for get_current_activity_header.
WARNING :: Thread-14 : Failed to access uri endpoint /status/sessions with error timed out
WARNING :: Thread-14 : PlexPy Pmsconnect :: Unable to parse XML for get_current_activity: 'NoneType' object has no attribute 'getElementsByTagName'.
DEBUG :: Thread-14 : PlexPy Monitor :: Unable to read session list.
WARNING :: Thread-14 : PlexPy Monitor :: Unable to get an internal response from the server, ping attempt 3.

**Comment by JonnyWong16** [comment]
You guys need to post full logs. Log snippets are not helpful at all.

Please follow the issues guidelines: https://github.com/JonnyWong16/plexpy#issues

**Comment by DKetch88** [comment]
Here is the full log I have.
[log.txt](https://github.com/JonnyWong16/plexpy/files/823254/log.txt)

**Comment by francesco2013** [comment]
19 days no answer ? I have the same damn problem .... everything runs but statistics (that is of course the main reason why I have installed plexpy) have the same issue of the other guys.


2017-03-25 23:07:43 - DEBUG   :: CP Server Thread-5 : PlexPy WebAuth :: Admin user '##########' logged into PlexPy.
2017-03-25 23:07:46 - ERROR   :: CP Server Thread-10 : No image input received.
2017-03-25 23:08:02 - WARNING :: CP Server Thread-9 : Unable to retrieve data for library_recently_watched.
2017-03-25 23:08:02 - WARNING :: CP Server Thread-4 : Unable to retrieve data for library_user_stats.
2017-03-25 23:08:12 - INFO    :: Thread-15 : PlexPy PlexTV :: Requesting users list refresh...
2017-03-25 23:08:12 - INFO    :: CP Server Thread-8 : Manual users list refresh requested.
2017-03-25 23:08:13 - INFO    :: Thread-15 : PlexPy PlexTV :: Users list refreshed.

**Comment by JonnyWong16** [comment]
Your issue is not the same as the others: https://github.com/JonnyWong16/plexpy/wiki/Frequently-Asked-Questions-(FAQ)#general-q7

**Comment by francesco2013** [comment]
I have made it work :) Thanks !

**Comment by xtianus79** [comment]
@francesco2013 how have you made it work

**Comment by xtianus79** [comment]
@JonnyWong16 my log looks exactly like @DKetch88

**Comment by xtianus79** [comment]
@JonnyWong16 @francesco2013 @DKetch88 @ChacalCH I have got it working again... 

Here are the steps I recreated... 

I figured something was wrong with my communication of plexpy and plex.  so I went to plexy to check settings. 

1.  In the webUI go to settings > Plex Media Server 

That check I had for my private ip was failing... I hit the button a few times and no go... 

2.  I went over to my plex webUI and checked some settings... settings > Remote Access 

My private IP was different by the last number... instead of a 2 it was a 3.  I copy and pasted the plex ip into the plexypy webui Plex Media Server and went back to plexypy and restared and everything works like a charm

Hope this helps.
